share/spice/yacht_specs/content.handlebars

```diff
@@ -0,0 +1,32 @@
+<!-- Div for the yacht specifications -->
+<div class='thumb'>
+    
+    <div class='image'>
+        <img src='{{result_photo}}' class='profile_photo' width=300px height=200px>
+    </div>
+    <div class='info'>
+       <h5 class='yachtName'>{{name}}</h5>
+        <table>
```

Comment:
Absolutely better this way. We've deleted the files and changed to the template.